src_erl/erlBridge/nerlNIF.erl

```diff
@@ -182,7 +183,8 @@ nerltensor_conversion({NerlTensor, Type}, ResType) ->
             encode -> Validated = validate_nerltensor_erl(NerlTensor),
                       if
                         Validated -> encode_nif(NerlTensor, BinType);
-                        true -> throw(nerl:string_format("encode failure due to incorrect dimension declaring X*Y*Z not equal to tensor data length! ~p ",[NerlTensor]))
+                        true -> io:format("Bad tensor format!!~n"), {<<>>, BinType}
```

Comment:
@halfway258 
Change to log error.
Maybe it's better: "Wrong NerlTensor format!"

Reply:
changed message